### PR TITLE
Clean up pyanaconda.installation_tasks

### PR DESCRIPTION
**Pull request by poncovka**
Simplify the code and remove everything we don't use.

### CODE REVIEW

**Comment by poncovka** [comment]
Rebased and tested.

File: pyanaconda/installation_tasks.py [hunk 1]
```diff
@@ -148,187 +119,93 @@ def status_message(self):
...
-                self._current_task_number = None
-                self._current_queue_number = None
-
-            # trigger the "completed" signals
-            self.completed.emit(self)
+    def _run(self):
+        """Run the task queue."""
+        for item in self._queue:
+            # start the item (TaskQueue/Task)
+            item.start()
```
Comment (VladimirSlavik) [inline pyanaconda/installation_tasks.py:189]: You start the item and connect the signals only later, I think? I don't see why this can work?

Comment (poncovka) [reply pyanaconda/installation_tasks.py:189]: It is a little messy, the signals are connected when you append an item to a queue.

**Comment by poncovka** [comment]
Btw. this code is not going to survive modularization, but it is simpler to keep it alive in its minimal form for now.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me, however, I wonder how is possible that you were able to remove so much of the code.

**Pull request merged**